cmds/report.c

```diff
@@ -184,6 +184,39 @@ static void print_and_delete(struct rb_root *root, bool sorted, void *arg,
 	}
 }
 
+static void print_csv_function(struct uftrace_report_node *node, void *unused)
+{
+        if (avg_mode == AVG_NONE) {
+                print_time_unit(node->total.sum);
+                pr_out(",");
+                print_time_unit(node->self.sum);
+		pr_out(",");
+                pr_out("  %10lu,  %-s\n", node->call, node->name);
+        }
+        else {
+                uint64_t time_avg, time_min, time_max;
+
+                if (avg_mode == AVG_TOTAL) {
+                        time_avg = node->total.avg;
+                        time_min = node->total.min;
+                        time_max = node->total.max;
+                }
+                else {
+                        time_avg = node->self.avg;
+                        time_min = node->self.min;
+                        time_max = node->self.max;
+                }
+                print_time_unit(time_avg);
+                pr_out(",");
+                print_time_unit(time_min);
+                pr_out(",");
+                print_time_unit(time_max);
+		pr_out(",");
```

Comment:
The overall indentations are made using whitespaces, but please fix them to "tabs" only.

Reply:
Fixed